From 9b6a090325f384d634eb4f9d5b1b8ce7ed0664de Mon Sep 17 00:00:00 2001
From: Huibin Hong <huibin.hong@rock-chips.com>
Date: Thu, 27 Apr 2017 09:13:53 +0800
Subject: [PATCH] Revert "fiq debugger: add resume for debug uart"

This reverts commit 0e7d751d6c004970348bdacb6961e06e7775cb92.

If enable no console suspend, it will do resume when printing log,
which causes problem. Remove resume of kernel, and add resume of
trust firmware or other power management code.

Change-Id: I3e8e704140134a6aad5c0eb2f14fde36fb108ad3
Signed-off-by: Huibin Hong <huibin.hong@rock-chips.com>
---
 drivers/soc/rockchip/rk_fiq_debugger.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/soc/rockchip/rk_fiq_debugger.c b/drivers/soc/rockchip/rk_fiq_debugger.c
index db1e784eea11..a569cde4998f 100755
--- a/drivers/soc/rockchip/rk_fiq_debugger.c
+++ b/drivers/soc/rockchip/rk_fiq_debugger.c
@@ -194,12 +194,6 @@ static void debug_flush(struct platform_device *pdev)
 		cpu_relax();
 }
 
-static int uart_dev_resume(struct platform_device *pdev)
-{
-	debug_port_init(pdev);
-	return 0;
-}
-
 #ifdef CONFIG_RK_CONSOLE_THREAD
 #define FIFO_SIZE SZ_64K
 static DEFINE_KFIFO(fifo, unsigned char, FIFO_SIZE);
@@ -442,7 +436,6 @@ void rk_serial_debug_init(void __iomem *base, phys_addr_t phy_base,
 	t->pdata.uart_init = debug_port_init;
 	t->pdata.uart_getc = debug_getc;
 	t->pdata.uart_putc = debug_putc;
-	t->pdata.uart_dev_resume = uart_dev_resume;
 #ifndef CONFIG_RK_CONSOLE_THREAD
 	t->pdata.uart_flush = debug_flush;
 #endif
-- 
2.35.3

